src/averageAges.js

```diff
@@ -1,61 +1,59 @@
 'use strict';
 
-/**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
- * @param {object[]} people
- * @param {number} century - optional
- *
- * @return {number}
- */
+function Men(people) {
+  const menPerson = people.filter(person => person.sex === 'm');
+
+  return menPerson;
+}
+
+function Women(people) {
+  const womenPerson = people.filter(person => person.sex === 'f');
+
+  return womenPerson;
+}
+
+function Mother(people) {
```

Comment:
does this function return a single object? or an array of them? that's why you should give appropriate names

Reply:
also, fix names of function over your could, consider there is no class in your code